pkgs/top-level/all-packages.nix

```diff
@@ -28434,6 +28434,8 @@ with pkgs;
 
   gotools = callPackage ../development/tools/gotools { };
 
+  gosimports = callPackage ../development/tools/gosimports { };
```

Comment:
```suggestion
```

The package should be placed on the new `pkgs/by-name/so/some-example/package.nix` hierarchy 😊

Reply:
Oh, I just noticed this folder. This hierarchy is really a good idea.